src/main/java/address/controller/PersonOverviewController.java

```diff
@@ -140,16 +183,10 @@ private void handleEditPerson() {
     }
 
     private boolean isEditSuccessful(ReadOnlyPerson oldPerson, ReadOnlyPerson newPerson) {
-        try {
-            modelManager.updatePerson(oldPerson, newPerson);
-            mainController.getStatusBarHeaderController().postStatus(
-                    new PersonEditedStatus(new Person(oldPerson), newPerson));
-            return true;
-        } catch (DuplicatePersonException e) {
-            mainController.showAlertDialogAndWait(AlertType.WARNING, "Warning", "Cannot have duplicate person",
-                    e.toString());
-            return false;
-        }
+        modelManager.updatePerson(oldPerson, newPerson);
+        mainController.getStatusBarHeaderController().postStatus(
+                new PersonEditedStatus(new Person(oldPerson), newPerson));
+        return true;
```

Comment:
if always returns true, inline the method or change the method name to something that tells what it does